doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -601,6 +601,13 @@ Wi-Fi samples
   * Added support for running the full stack on the Thingy:91 X.
      This is a special configuration that uses the nRF5340 as the host chip instead of the nRF9151.
 
+Nordic Immutable Secure Bootloader |NSIB|
+-----------------------------------------
+
+* :ref:`bootloader` Changed allowed firmware info offsets:
+
+  * :ref:`doc_fw_info` Can now be placed at offset 0x600. Note that this offset cannot be used for any applications with |NSIB| compiled before this change.
```

Comment:
Should this be under samples? Wouldn't it fit better under bootloaders and DFU section? And if that's preferable, then we could include it under "updated" subsection:

```
* Updated: 

  * The procedure for signing the application image built for booting by MCUboot in direct-XIP mode with revert support.
    Now, the Intel-Hex file of the application image automatically receives a confirmation flag.
  * The allowed offset for :ref:`doc_fw_info` in the :ref:`bootloader`.
    It can now be placed at offset 0x600, however, it cannot be used for any applications with |NSIB| compiled before this change.
```